images/cfw/etc/init.d/S01logging

```diff
@@ -2,6 +2,8 @@
 #
 # Start logging
 
+. /usr/share/x1plus/functions.sh
```

Comment:
I think this is a good improvement.  One thing I am thinking is that we probably should be more disciplined about where on the filesystem we are dropping stuff.  I had originally been thinking of migrating a bunch of our tools to `/opt/x1plus/lib` and `/opt/x1plus/bin` and such.  I guess I could be persuaded that `/usr/share/x1plus` is a reasonable place too (or `/usr/x1plus/libexec/functions.sh` or something?).

I'm not sure I really have it in me to block this PR on that, but I think the day of reckoning for our paths is coming sooner rather than later.  So if you feel like it it might be nice for this PR to start whatever the New System of Paths is going to be.  My vote for this one is `/opt/x1plus/libexec/functions.sh`.